systemtests/utils/vagrantnode.go

```diff
@@ -15,34 +15,114 @@ limitations under the License.
 
 package utils
 
+import (
+	"fmt"
+	"io/ioutil"
+
+	"golang.org/x/crypto/ssh"
+)
+
 // VagrantNode implements a node in vagrant testbed
 type VagrantNode struct {
-	Name    string
-	NodeNum int
+	Name   string
+	client *ssh.Client
+}
+
+//NewVagrantNode intializes a node in vagrant testbed
+func NewVagrantNode(name string, nodeNum int, port, privKeyFile string) (*VagrantNode, error) {
```

Comment:
good point, it was there in earlier version of VagrantNode but later I decided to clean it up. Will fix this.